### ISSUE TITLE
Travis deploy prompts for deploy key

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/haxe.org#163]
```
Enter passphrase for deploy_key: 
No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated
```

**Comment by andyli** [comment]
The Travis log said
```
bad decrypt
140170419500704:error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt:evp_enc.c:539:
Agent pid 2405
```

I guess there is an update on Travis's side invalidated our encrypted key.
@jasononeil, Would you try to run `travis encrypt` again and replace the the encrypted key?

**Comment by andyli** [comment]
@jasononeil please follow up in https://github.com/travis-ci/travis-ci/issues/5442

**Comment by andyli** [comment]
Fixed in 2937079b22d36bef1bf1d57f0c1028f02e422d8d
